app/brave_generated_resources.grd

```diff
@@ -479,6 +479,10 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_HELP_TIPS_SHOW_BRAVE_WAYBACK_MACHINE_PROMPT" desc="The label describing the 'wayback machine' toggle option">
         Show Wayback Machine prompt on 404 pages
       </message>
+      <!-- Sync Infobar -->
+      <message name="IDS_BRAVE_SYNC_V1_DEPRECATED_INFOBAR_MESSAGE" desc="Message text shown to users who used a previous version of Brave Sync and need to learn about a new version coming soon.">
+        A new Brave Sync is coming. Get ready for the upgrade.
```

Comment:
I don't see this message telling user that sync v1 will soon be no longer available. Or will user see that in https://brave.com/sync-v2-is-coming?

Reply:
Yeah it should be there I suppose. But I'll bring up the question as to whether we should show this to users who are currently _syncing_, i.e. have manually turned the sync v1 flag on.

Reply:
Made the following changes based on feedback:

- Updated the message according to @jsecretan's suggestion:
> A new Brave Sync is coming and will affect your setup. Get ready for the upgrade.

- This infobar only shows to users who have re-enabled sync v1 via the flag.